### PR TITLE
Add README badges for the docker image

### PR DESCRIPTION
**Pull request by nstapelbroek**
Hi!

I was looking for the docker image on the hub and it took a couple of minutes.
So maybe we can add some badges to link to the image on docker hub?

### CODE REVIEW

**Comment by nstapelbroek** [comment]
There's [more badges available on the internet](https://gist.github.com/tterb/982ae14a9307b80117dbf49f624ce0e8), but not sure how much you want to pimp this readme 🙈

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@ Discord Soundbot
 ================
 
 [<img title="Join us on Discord" src="https://discordapp.com/api/guilds/236732117524938754/widget.png?style=shield">](https://discord.gg/JBw2BNx)
+[![Docker Pulls](https://img.shields.io/docker/pulls/markokajzer/discord-soundbot.svg?style=shield)](https://hub.docker.com/r/markokajzer/discord-soundbot/tags/)
+[![](https://images.microbadger.com/badges/image/markokajzer/discord-soundbot.svg?style=shield)](https://microbadger.com/images/markokajzer/discord-soundbot)
```
Comment (markokajzer) [inline README.md:6]: Don't really fancy this one. Which uber :neckbeard: would be interested in how many layers there are? 🤓 

What do you think about removing this one?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,9 @@ Discord Soundbot
...
+[![license](https://img.shields.io/github/license/markokajzer/discord-soundbot.svg?style=shield)](https://opensource.org/licenses/MIT)
```
Comment (markokajzer) [inline README.md:7]: I don't think most people are interested in this. Maybe put a sentence in the README in the Contributing section instead?

**Comment by nstapelbroek** [comment]
I've  updated the PR. I ended up aligning the badges in the middle together as creating multiple lines might look better once there are more badges. 

Before:
Screenshot 2020-07-04 at 10 55 43

After: 
Screenshot 2020-07-04 at 11 01 20

Some other changes include updating the current discord join button and implementing the support button to match the style requested. I assume the download counter is not working nicely because people there aren't any artefacts to download besides the source code which is probably cloned as the readme suggests. It would be cool to have some statistics of bot running "in the wild" but this seems a bit out of scope for this PR.

**Comment by nstapelbroek** [comment]
With the buttons aligned like that I imagine a space for a short demo video or logo 😍

**Comment by markokajzer** [comment]
This looks so tight man!! Thank you so much 😍

**Pull request merged**